### PR TITLE
Homework Tools #2

### PR DESCRIPTION
**Pull request by Kasperice**

### CODE REVIEW

File: src/functions.cpp [hunk 1]
```diff
@@ -1,10 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (unsigned int i = 0; i < numbers.size(); ++i) {
```
Comment (Marcinati) [inline src/functions.cpp]: I recommend to use size_t here istead of unsigned int, that's why:
https://en.cppreference.com/w/cpp/types/size_t

**Review by Marcinati — APPROVED** [review]
Everything looks good. I recommend to change one thing.

File: src/functions.cpp (hunk 1 above)
Comment (Kasperice) [reply src/functions.cpp]: Thank you, I changed that

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -Wall -Wextra -pedantic -Werror -o main
+	g++ -std=c++17 src/*.cpp -Wall -Wextra -pedantic -Werror -Ilib -o main
```
Comment (ziobron) [inline Makefile]: -Iinc not -Ilib. This code will not compile

Comment (ziobron) [reply Makefile]: ok, already fixed ;)

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**